### DISCUSSION TITLE
How to iterate over slides with x-for with Glide?

### DISCUSSION THREAD

**Discussion by nick-dolan** [alpinejs/alpine#3342] [1. Help] [answered]
I'm trying this example https://alpinejs.dev/component/glide but with `x-for` to get slides. It doesn't seem work.

```html
<ul class="glide__slides">
  <template
      x-for="item in 5"
      :key="item">
    <li class="glide__slide">
      <img class="w-full" src="https://picsum.photos/800/400?random=1" alt="placeholder image">
    </li>
  </template>
</ul>
```

**Comment by ekwoka** [comment, marked as answer]
What does "doesn't seem to work" mean? What are you getting that you're not expecting? What are you expecting by not getting? Are there errors?

Just based off of this little info, I'd assume it's that glide is initializing before the for loop has rendered the elements, so you should delay glide initializing with `nextTick`

**Reply by nick-dolan** [reply]
`nextTick` works, thanks
